test/release/index.js

```diff
@@ -82,18 +42,6 @@ const pages = [
         "key": "heatmap-layer",
         "title": "Add a heatmap layer"
     },
-    {
-        "key": "add-terrain",
-        "title": "Add Terrain"
-    },
-    {
-        "key": "atmospheric-sky",
-        "title": "Atmospheric Sky"
-    },
-    {
-        "key": "free-camera-point",
-        "title": "Free Camera Point"
-    },
```

Comment:
Do you know where the free camera API is now covered?

Reply:
The free camera api is included in animate a camera along path: https://docs.mapbox.com/mapbox-gl-js/example/free-camera-path (and 3d playground)

Reply:
Thanks for confirming!